src/pluginDef.ts

```diff
@@ -13,8 +13,8 @@ import { ProfileConstants } from "./api/ProfileConstants";
 
 const pluginDef: IImperativeConfig = {
     commandModuleGlobs: ["**/cli/*/*.definition!(.d).*s"],
-    pluginLifeCycle: __dirname + "/LifeCycleForSample",
     pluginSummary: "Zowe CLI sample plug-in",
+    pluginLifeCycle: "LifeCycleForSample",
```

Comment:
I haven't tested this but I'm curious if we need to reference this file from its transpiled location (`lib/LifeCycleForSample`)

Reply:
tests were breaKing without this line @t1m0thyj

Reply:
It seems safer to keep the absolute path like we had before:
`pluginLifeCycle: __dirname + "/LifeCycleForSample",`
But if it works with a relative path then maybe that's fine too? 😋